tests/core/test_structure.py

```diff
@@ -114,7 +114,6 @@ def test_as_dataframe(self):
 
     def test_equal(self):
         struct = self.struct
-        assert struct == struct
         assert struct == struct.copy()
```

Comment:
these lines are important to keep in `__eq__` tests. we want to make sure the magic method returns `True` as expected

Reply:
Thanks, I didn't realize that!